Add ibm_is_instance_dashboard, age & detail report. Closes #4 (#19)
turbot/steampipe-mod-ibm-insights: dashboards/compute/docs/compute_instance_detail.md

This dashboard answers the following questions for each instance:

- How is the instance configured?
- What tags are applied?
- How many vCPU cores are present on each instance?

> - Which boot volume, data volume, disks, security groups and network interfaces is it attached to?

Reviewer: ```suggestion
- Which boot volume, data volume, instance storage disks, and network interfaces are attached to?
```
